src/app/getting-started/getting-started.component.html

```diff
@@ -0,0 +1,96 @@
+<div id="overview" class="container content padding-top-15" *ngIf="showGettingStarted">
+  <div class="row">
+    <h1 class="getting-started-title">Getting started in OpenShift.io</h1>
+    <p class="getting-started">
+      Welcome to OpenShift.io! To get started you will need to create a username and connect
+      your GitHub and OpenShift.com accounts.
+    </p>
+    <p class="padding-bottom-15">
+      The fields marked with <span class="required-pf">*</span> are required.
+    </p>
+    <form role="form">
+      <div class="row">
+        <div class="col-md-1">
+          <h2 class="circle">1</h2>
+        </div>
+        <div class="col-md-11 padding-left-0">
+          <h2>Select a username</h2>
+          <div class="form-group">
+            <label for="username" class="control-label required-pf">Username</label>
+            <div [ngClass]="{'has-error': usernameInvalid}">
+              <div class="col-md-5 padding-left-0">
+                <input type="text" class="form-control" id="username" name="username" maxlength="62" required
+                       [disabled]="registrationCompleted"
+                       [(ngModel)]="username"
+                       (change)="updateUsername()">
+              </div>
+              <div class="col-md-7 padding-left-0">
+                <span class="help-block padding-0">
+                Username must be less than 63 characters and can't begin with a dash or underscore. They may contain
+                lowercase letters, numbers, dashes, or underscores.
+              </span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-1">
+          <h2 class="circle">2</h2>
+        </div>
+        <div class="col-md-11 padding-left-0">
+          <h2>Connected accounts</h2>
+          <p class="getting-started">
+            This is where we explain that the user must connet their GitHub and OpenShift accounts to OpenShit.io.
+            We also explain all the benefits that come with connecting accounts.
+          </p>
+          <label class="margin-bottom-10" *ngIf="!(gitHubLinked && openShiftLinked)">
+            Select accounts to authorize
+          </label>
+          <label class="margin-bottom-15 required-pf" *ngIf="gitHubLinked && openShiftLinked">
```

Comment:
New to Angular v4 is the "else" part of the "ngIf" statement. It may simplify this section.